zio-http/shared/src/main/scala/zio/http/endpoint/Endpoint.scala

```diff
@@ -575,7 +575,7 @@ final case class Endpoint[PathInput, Input, Err, Output, Auth <: AuthType](
       output = ((HttpCodec.content[Output2] ++ StatusCodec.status(status)) ?? doc) | self.output,
       error,
       codecError,
-      Doc.empty,
```

Comment:
I didn't understand why using `myEndpoint.out[...](status = ..., doc = ...)` would reset the doc to an empty one. Probably a typo